### DISCUSSION TITLE
Homepage matching sidebar gradient colors

### DISCUSSION THREAD

**Discussion by nuexq** [zen-browser/desktop#3943] [Ideas]
I see sometimes when I load a tab I see the that the browser match the sidebar gradient like this
![image](https://github.com/user-attachments/assets/6f91d581-0686-4219-8565-698dfaa5e526)
It would be fantastic if the blank page match the sidebar colors like the photo. or making the blank page option to be like the photo instead of a grey page

**Comment by 12th-devs** [comment]
@No0ne003 you can, go to "Home" in setting and change the drop downs to Blank Page

**Reply by nuexq** [reply]
the blanks looks like this as i say just a grey page it will be batter to match sidebar colors
![image](https://github.com/user-attachments/assets/2b7e0229-2c69-450d-ac97-c96d39c36d9c)

**Reply by 12th-devs** [reply]
It's an about config setting that you have to change. I don't remember off of the top of my head but you can search around on here and find it pretty easily.

**Reply by nuexq** [reply]
I can't find it

**Comment by nuexq** [comment]
I think it's added on the last update maybe cuz it works now
